Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -308,6 +308,26 @@
           DefaultPrototype: Beaker
           ExamineName: Glass Beaker
 
+- type: entity
+  id: SeedExtractorMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: seed extractor machine board
+  description: A machine printed circuit board for a seed extractor
+  components:
+    - type: Sprite
+      state: service
+    - type: MachineBoard
+      prototype: SeedExtractor
+      # See: https://github.com/vgstation-coders/vgstation13/blob/e9a806f30b4db0efa2a68b9eb42e3120d2321b6a/code/modules/hydroponics/seed_extractor.dm#L21
+      requirements:
+        Manipulator: 2
+        MatterBin: 1
+        Laser: 1
+        ScanningModule: 1
```

Comment:
if only manipulators have any effect on the machine, then the other parts should be removed and replaced with either generic materials or cables and such (outlined by #11997). Additionally, since the machine is powered, you could add a capacitor and give seed extractors the `UpgradePowerDrawComponent`.

Reply:
Done, now it's just 2 manips and 1 capacitor, plus 2 lv cable and 1 glass.